src/tests/access/test_dual_accesscontrol.cairo

```diff
@@ -1,22 +1,16 @@
 use openzeppelin::access::accesscontrol::DEFAULT_ADMIN_ROLE;
 use openzeppelin::access::accesscontrol::dual_accesscontrol::DualCaseAccessControl;
 use openzeppelin::access::accesscontrol::dual_accesscontrol::DualCaseAccessControlTrait;
-use openzeppelin::access::accesscontrol::interface::IACCESSCONTROL_ID;
-use openzeppelin::access::accesscontrol::interface::IAccessControlCamelDispatcher;
-use openzeppelin::access::accesscontrol::interface::IAccessControlCamelDispatcherTrait;
-use openzeppelin::access::accesscontrol::interface::IAccessControlDispatcher;
-use openzeppelin::access::accesscontrol::interface::IAccessControlDispatcherTrait;
-use openzeppelin::tests::mocks::accesscontrol_mocks::{
-    CamelAccessControlMock, SnakeAccessControlMock, CamelAccessControlPanicMock,
-    SnakeAccessControlPanicMock
+use openzeppelin::access::accesscontrol::interface::{
+    IACCESSCONTROL_ID, IAccessControlDispatcher, IAccessControlDispatcherTrait,
+    IAccessControlCamelDispatcher, IAccessControlCamelDispatcherTrait
```

Comment:
Good call consolidating these, though we proposed this general rule with keeping imports up to three lines [here](https://github.com/OpenZeppelin/cairo-contracts/issues/696#issuecomment-1768198706). IMO it makes sense to group a dispatcher and its associated trait within a single `use` declaration

Reply:
Removed consolidation. Due to the long module path ungrouped imports take fewer lines and look nicer